src/index.js

```diff
@@ -622,6 +622,7 @@ function createConfig(options, entry, format, writeMeta) {
 									fs.writeFile(
 										getNameCachePath(),
 										JSON.stringify(nameCache, null, 2),
+										() => {},
```

Comment:
same as above, does this change the behavior?

Reply:
same deal, was a typescript warning